### PR TITLE
Add select-all and select-none buttons to TimeseriesSearch

### LINKED ISSUE #274: Add select all, select none to i.e. timeseries search
On the timeseries search, we may get a list of N time series that match our search query. It would be really nice if we had an option to have `select all` and `select none` buttons.

### PR DESCRIPTION
**Pull request by follan**
Fixes #274

### CODE REVIEW

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx [hunk 1]
```diff
@@ -34,8 +35,25 @@ const CenteredSpin = styled(Spin)`
   }
 `;
 
+const ButtonRow = styled.div`
+  margin-top: 5px;
+`;
+
+const SelectAllButton = styled((props: NativeButtonProps) => (
```
Comment (mrtraser) [inline src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Do we need styled component here if we don't provide any styles to it?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-282.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx (hunk 1 above)
Comment (follan) [reply src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Removed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-282.gearboxjs.preview.cogniteapp.com

**Pull request merged**